02week/ticTacToe.js

```diff
@@ -24,39 +24,80 @@ function printBoard() {
 }
 
 function horizontalWin() {
-  // Your code here
+  return board[0].every(square => square === playerTurn) ||
```

Comment:
Excellent formatting and use of the `.every()` js method.

![excellent](https://media.giphy.com/media/3oEdv31lb6MQv2XNfO/giphy.gif)